assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -164,10 +163,14 @@ const baseActions = {
 			invariant( timestamp, 'A timestamp is required.' );
 		},
 		function* ( timestamp ) {
-			yield {
-				type: CACHE_LAST_DISMISSED_AT,
-				payload: { timestamp },
-			};
+			const registry = yield getRegistry();
+
+			yield registry
```

Comment:
@zutigrm @tofumatt This will not work the same as before where `yield` would behave as `await`. This probably isn't a big deal in this instance but it is technically incorrect. See https://github.com/google/site-kit-wp/issues/6117

It doesn't need to block the release but it should be fixed.